docs/src/pages/components/menus/menus.md

```diff
@@ -67,6 +67,12 @@ Use a different transition.
 
 {{"demo": "pages/components/menus/FadeMenu.js"}}
 
+## Context Menu
+
+The `ContextMenu` wraps the area in which you'd like right-click functionality. Just add the tag `<ContextMenu></ContextMenu>` around the element and your custom context menu will be available anywhere within that element.
```

Comment:
Avoid "just" when possible

```suggestion
The `ContextMenu` wraps the area in which you'd like right-click functionality. add the tag `<ContextMenu></ContextMenu>` around the element and your custom context menu will be available anywhere within that element.
```

Reply:
Also, I'm not sure that the description matches the content of the demo.

Reply:
I will rewrite the description.